config/shared/call.py

```diff
@@ -36,5 +37,12 @@ def call(*args, capture_output=False, cwd=None, allowed_return_codes=(0, )):
     result = subprocess.run(
         args, capture_output=capture_output, text=True, cwd=cwd)
     if result.returncode not in allowed_return_codes:
-        abort(result.returncode)
+        if capture_output:
+            abort_text = textwrap.defent(f'''
```

Comment:
Is this a typo?

```suggestion
            abort_text = textwrap.dedent(f'''
```

Reply:
Yes it is a typo. So this code has never been run. Shame on me.
Thank you for spotting it.